documentation/MIGRATION_GUIDE.md

```diff
@@ -0,0 +1,186 @@
+
+## Table of contents
+
+* [Prerequisites](#prerequisites)
+* [General Changes](#general-changes)
+    * [Authentication](#authentication)
+    * [Customized Policy](#customized-policy)
+    * [Custom HTTP Client](#custom-http-client)
+    * [Error Handling](#error-handling)
+    * [Long Running Operations](#long-running-operations)
+    * [Pagination](#pagination)
+
+## Prerequisites
+
+Golang 1.13 or above.
+
+## General Changes
+
+The latest Azure SDK For Go for management modules is using the [Go Modules](https://github.com/golang/go/wiki/Modules) to manage the dependencies. We ship every RP as an individual module to create a more flexible user experience.
+
+The important breaking changes are listed in the following sections:
+
+### Authentication
+
+In old version (`services/**/mgmt/**`), `autorest.Authorizer` is used in authentication process.
+
+In new version (`sdk/**/arm**`), in order to provide a unified authentication based on Azure Identity for all Azure Go SDKs, the authentication mechanism has been re-designed and improved to offer a simpler interface.
+
+To the show the code snippets for the change:
+
+**In old version (`services/**/mgmt/**`)**
+
+```go
+authorizer, err := adal.NewServicePrincipalToken(oAuthToken, "<ClientId>", "<ClientSecret>", endpoint)
+```        
+
+**Equivalent in new version (`sdk/**/arm**`)**
+
+```go
+credential, err = azidentity.NewClientSecretCredential("<TenantId>", "<ClientId>", "<ClientSecret>", nil)
+```
+
+For detailed information on the benefits of using the new authentication classes, please refer to [this page](https://github.com/Azure/azure-sdk-for-go/blob/main/sdk/azidentity/README.md)
+
+### Customized Policy
+
+Because of adopting Azure Core which is a shared library across all Azure SDKs, there is also a minor change regarding how customized policy in configured.
+
+In old version (`services/**/mgmt/**`), we use the `(autorest.Client).Sender`, `(autorest.Client).RequestInspector` and `(autorest.Client).ResponseInspector` properties in `github.com/Azure/go-autorest/autorest` module to provide customized interceptor for the HTTP traffic.
+
+In new version (`sdk/**/arm**`), we use `(armcore.ConnectionOptions).PerCallPolicies` and `(armcore.ConnectionOptions).PerRetryPolicies` in `github.com/Azure/azure-sdk-for-go/sdk/armcore` module instead to inject customized policy to the pipeline.
+
+### Custom HTTP Client
+
+Similar to the customized policy, there are changes regarding how the custom HTTP client is configured as well. You can now use the `(armcore.ConnectionOptions).HTTPClient` option in `github.com/Azure/azure-sdk-for-go/sdk/armcore` module to use your own implementation of HTTP client and plug in what they need into the configuration.
+
+**In old version (`services/**/mgmt/**`)** you cannot customize the HTTP client.
```

Comment:
This is not true.  In track 1 you'd set the `client.Sender` field to a custom HTTP client.

Reply:
Done